conf/configuration.yaml

```diff
@@ -73,6 +73,7 @@ arlas-index:  ${ARLAS_ELASTIC_INDEX:-.arlas}
 elastic-enable-ssl: ${ARLAS_ELASTIC_ENABLE_SSL:-false}
 elastic-credentials: ${ARLAS_ELASTIC_CREDENTIALS:-user:password}
 elastic-compress: ${ARLAS_ELASTIC_COMPRESS:-true}
+elastic
```

Comment:
is something missing Or it's set to undefined to keep using the deprecated config attributes?

Reply:
oops! it's a mistake

Reply:
You have my approval